native/src/main/java/io/ballerina/stdlib/email/server/EmailListener.java

```diff
@@ -119,13 +115,8 @@ protected void addService(BObject service) {
         }
     }
 
-    private void invokeAsyncCall(BObject service, String methodName, StrandMetadata metadata, Object arg) {
-        ObjectType serviceType = (ObjectType) TypeUtils.getReferredType(TypeUtils.getType(service));
-        if (serviceType.isIsolated() && serviceType.isIsolated(methodName)) {
-            runtime.call(service, methodName, null, metadata, null,  arg);
-        } else {
-            runtime.startNonIsolatedWorker(service, methodName, null, metadata, null, arg);
-        }
+    private void invokeAsyncCall(BObject service, String methodName, Object arg) {
```

Comment:
Do we need this seperate method. Anyway it is not async call. We can simply use runtime.call(service, methodName, arg) in other places;